gatsby_fp/gatsby-config.js

```diff
@@ -38,6 +38,8 @@ module.exports = {
             notable_courses: String
             school: String
             ses_department: String
+            alt_publications: String
+            notable_publications: String
```

Comment:
I believe this change is unrelated to the caching issue, if yes, I think this should be in a separate PR.

Reply:
I think this code has already been merged to master... this branch was probably made before that PR (#50) was merged.  I'll remove this code or manually resolve the merge conflicts once our testing is done so we don't get any unrelated issues during testing caused by removing this.

Reply:
When a code change is already in master but you have some feature branch that was created before that point, and you want the feature branch to also contain that code change, the best way to achieve that is to merge master into the feature branch, not manually commit the same code to the feature branch. If said merge causes conflicts, resolve them during the merge.

When this particular change was committed to this feature branch, the commit message was "Fixed merge conflicts from master" which suggests that it was done properly. On bitbucket, this would not show up as part of the full diff when looking at the feature branch PR, but maybe Github is different.